src/olympia/lib/settings_base.py

```diff
@@ -763,14 +775,28 @@ def JINJA_CONFIG():
             'js/common/keys.js',
 
             # jQuery UI
-            'js/lib/jquery-ui/core.js',
-            'js/lib/jquery-ui/position.js',
-            'js/lib/jquery-ui/widget.js',
-            'js/lib/jquery-ui/mouse.js',
-            'js/lib/jquery-ui/menu.js',
-            'js/lib/jquery-ui/autocomplete.js',
-            'js/lib/jquery-ui/datepicker.js',
-            'js/lib/jquery-ui/sortable.js',
+            'js/node_lib/ui/version.js',
+            'js/node_lib/ui/data.js',
+            'js/node_lib/ui/disable-selection.js',
+            'js/node_lib/ui/ie.js',
+            'js/node_lib/ui/keycode.js',
+            'js/node_lib/ui/escape-selector.js',
+            'js/node_lib/ui/labels.js',
+            'js/node_lib/ui/jquery-1-7.js',
+            'js/node_lib/ui/plugin.js',
+            'js/node_lib/ui/safe-active-element.js',
+            'js/node_lib/ui/safe-blur.js',
+            'js/node_lib/ui/scroll-parent.js',
+            'js/node_lib/ui/focusable.js',
+            'js/node_lib/ui/tabbable.js',
+            'js/node_lib/ui/unique-id.js',
+            'js/node_lib/ui/position.js',
+            'js/node_lib/ui/widget.js',
+            'js/node_lib/ui/widgets/mouse.js',
+            'js/node_lib/ui/widgets/menu.js',
+            'js/node_lib/ui/widgets/autocomplete.js',
+            'js/node_lib/ui/widgets/datepicker.js',
+            'js/node_lib/ui/widgets/sortable.js',
```

Comment:
More files, but the same content.  core.js was [split](https://github.com/jquery/jquery-ui/blob/1.12.1/ui/core.js) into 15 different files.  I did a quick search for each of the files, and they all _appeared_ to be used, or at least referenced in our js - working out whether all of _our_ js is still used is another issue.